tests/auth_tests/test_tokens.py

```diff
@@ -27,25 +27,47 @@ def test_10265(self):
         tk2 = p0.make_token(reload)
         self.assertEqual(tk1, tk2)
 
-    def test_timeout(self):
```

Comment:
We should move existing tests to a separate class (or even a separate file) and add tests for deprecation, e.g.
```python
class PasswordResetTimeoutDaysTests(TestCase):
    msg = 'The PASSWORD_RESET_TIMEOUT_DAYS setting is...'

    @ignore_warnings(category=RemovedInDjango40Warning)
    def test_timeout(self):
        ...

    def test_override_settings_warning(self):
        with self.assertRaisesMessage(RemovedInDjango40Warning, self.msg):
             ...

    def test_settings_init_warning(self):
        settings_module = ModuleType('fake_settings_module')
        settings_module.PASSWORD_RESET_TIMEOUT_DAYS = 2
        sys.modules['fake_settings_module'] = settings_module
        try:
            with self.assertRaisesMessage(RemovedInDjango40Warning, self.msg):
                Settings('fake_settings_module')
        finally:
            del sys.modules['fake_settings_module']    
```